docs/examples.rst

```diff
@@ -1,4 +1,10 @@
 Examples
 ========
 
-TODO: document here
+You shall take a look at folium's example gallery_.
+
+.. _gallery: http://nbviewer.jupyter.org/github/python-visualization/folium/tree/master/examples/
+
+If this is not enough, you may find fancier examples here_.
+
+.. _here: http://nbviewer.jupyter.org/github/ocefpaf/folium_notebooks/tree/master/
```

Comment:
Yeah.  That division is important.  Still, I want to get the cookbook away from my personal account and into something more "official," where others will be more likely to contribute.